src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/TempPE/TempPEBuildManager.DesignTimeInputsDelta.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.TempPE
+{
+    internal partial class TempPEBuildManager
+    {
+        internal class DesignTimeInputsDelta
+        {
+            public bool ShouldCompile { get; set; } = true;
+            public ImmutableArray<string> AddedItems { get; set; } = ImmutableArray<string>.Empty;
+            public ImmutableArray<string> RemovedItems { get; set; } = ImmutableArray<string>.Empty;
+            public ImmutableArray<string> AddedSharedItems { get; set; } = ImmutableArray<string>.Empty;
+            public ImmutableArray<string> RemovedSharedItems { get; set; } = ImmutableArray<string>.Empty;
+            public IImmutableDictionary<NamedIdentity, IComparable> DataSourceVersions { get; set; } = ImmutableSortedDictionary<NamedIdentity, IComparable>.Empty;
+            public bool NamespaceChanged { get; internal set; }
+            public string OutputPath { get; internal set; }
```

Comment:
These setters are already internal. Why mark them in particular?

Reply:
That's the default behavior for the "Add Property" code fixer, must have forgotten to clean them up

Reply:
Oops! I should have closed this review, #4740 is the current one. I'm breaking this up into pieces to make reviewing easier. The "ditto" comment is still valid in that branch though, so thanks :D